src/workerd/server/workerd-api.c++

```diff
@@ -342,9 +343,8 @@ CompatibilityFlags::Reader WorkerdApi::getFeatureFlags() const {
   return *impl->features;
 }
 jsg::JsContext<api::ServiceWorkerGlobalScope> WorkerdApi::newContext(jsg::Lock& lock) const {
-  jsg::NewContextOptions options{
-    .newModuleRegistry = impl->tryGetModuleRegistry(),
-  };
+  jsg::NewContextOptions options{.newModuleRegistry = impl->tryGetModuleRegistry(),
+    .enableWeakRef = getFeatureFlags().getJsWeakRef()};
```

Comment:
clang-format will format this a bit better if you leave a trailing comma after `.getJsWeakRef(),`.